appveyor.yml

```diff
@@ -1,7 +1,13 @@
 init:
   - git config --global core.autocrlf input
 build_script:
-  - cmd: build.cmd BuildApp
+  - cmd: build.cmd CreatePackages
   - cmd: build.cmd UnitTests
   - cmd: build.cmd ConventionTests
 test: off
+nuget:
+  account_feed: true
+  project_feed: true
+  disable_publish_on_pr: true
```

Comment:
I'm not 100% clear on this value - if this is disabled, it would mean packages aren't created for PRs - which is the inspiration for this change. Or am I mis-understanding something?